src/service.js

```diff
@@ -0,0 +1,316 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
+
+import Proto from 'uberproto';
+import mongoose from 'mongoose';
+import filter from 'feathers-query-filters';
+import errors from 'feathers-errors';
+import errorHandler from './error-handler';
+
+// Use native promises
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
```

Comment:
I think that the consistency with other adapters is more valuable.  We definitely need to agree upon a standard for organizing and using services for the generators..